### PR TITLE
Complete Project Conference with html scss

### PR DESCRIPTION
**Pull request by heintayzar-hm**
Use Scss for css.
<a href="https://github.com/heintayzar-hm/linters-config-scss/pull/1">Changed Linters</a>
Add animations to cards links.
Make responsive.
Make the featured speakers section rendered by js.
Make mobile menu for pages

<a href="https://www.loom.com/share/6028ca906a744730b814ee614aa6c20e" target= "_blank" >Loom Description</a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,101 @@
...
+Free feel to use the project as you like
+
+
+### Run tests
+You can use vscode extension or just run the index file. (I have yet to set up local server)
+
+
+
+### Deployment
+
```
Comment (mckent05) [inline README.md]: - As part of the project requirements, please deploy your website on GitHub pages and include a link to the live version in your README 👍 👍

File: public/pages/index.html [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    <header>
+        <div class="bar hidden-if-small">
+            <ul>
+                <li><a href="#"><img id="facebook" src="../resources/facebook.svg"></a></li>
+                <li><a href="#"><img id="twitter" src="../resources/twitter.svg"></a></li>
+                <li>English</li>
+                <li>My Page</li>
+                <li>Logout</li>    
+            </ul>
+        </div>
```
Comment (mckent05) [inline public/pages/index.html]: - To improve the HTML semantics of your site and for better accessibility, please include this in your `nav` tag 🙏 🙏

File: public/pages/index.html [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+                <li><a class="link" data-replace="About" href='./about-me.html'><span class="link-text">About</span></a></li>
+                <li><a class="main-program-link link" data-replace="Program" href='#main-program'><span class="link-text">Program</span></a></li>
+                <li><a class="link" data-replace="Join" href='#'><span class="link-text">Join</span></a></li>
+                <li><a class="link" data-replace="Sponsor" href='#footer'><span class="link-text">Sponsor</span></a></li>
+                <li><a class="link" data-replace="News" href='#'><span class="link-text">News</span></a></li>
+            </ul>
+            <button type="button" class="hidden-if-small">
+                <span>Campaign</span>
+            </button>
+        </nav>
```
Comment (mckent05) [inline public/pages/index.html]: - Good job on the work done so far 👏 👏 . But there are some differences between the nav in your site and the one in the design. Please pay attention to the pictures attached below 👇 . Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your design:
-- Please pay attention to the following
capstone navbar

## For the top navbar with the social links:
 - There is a right margin on the right which isn't included in your design 🔰 
 - Please ensure that the social links match the ones in the design (color, font weight, e.t.c)
 - Please ensure that the height of the top navbar is almost the same as the one in the design 👍 👍 .

## For the white navbar containing the links:
 - The margins on the left side of the navbar should be the same as the margin on the right side. 😄 😄 

Design Template:
navbar behance

File: public/pages/about-me.html [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+        <header>
+            <div class="bar hidden-if-small">
+                <ul>
+                    <li><a href="#"><img id="facebook" src="../resources/facebook.svg"></a></li>
+                    <li><a href="#"><img id="twitter" src="../resources/twitter.svg"></a></li>
+                        <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>    
+                </ul>
+            </div>
```
Comment (mckent05) [inline public/pages/about-me.html]: - As I suggested on the home page, please include this in your `nav` tag to improve your site's HTML semantics and accessibility 👍 👍

File: public/pages/about-me.html [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+        </header>
+    <section class="about">
+        <h1 class="about-text-1"><span class="about-text-1-span hidden-if-large">"Hello! Sharing Fans"</span><span class="main-text">KPOP Conference 2022</span></h1>
+        <p class="about-text-2">The Korea Remixed K-pop Conference explores the historic, cultural, and political significance of South Korea&apos;s signature music. The conference will feature panels with IU experts, visiting scholars, and IU graduate and undergraduate students.</p>
+        <div class="about-contact">
+            <span class=".about-contact-span">Please Contact us per Email for any further questions</span>
+            <span class=".about-contact-span">about KPOP conference 2022!!</span>
+            <span class=".about-contact-span">ccsumit2015@cckorea.org</span>
+        </div>
+    </section>
```
Comment (mckent05) [inline public/pages/about-me.html:66]: - The color of the page header is different from the design template. Please ensure you fix this 😄 . Remember it is essential that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
Your project 👇:
about capstone

File: public/pages/about-me.html [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+    <section class="past-projects">
+        <h3 class="past-projects-h3">See Our Pasts</h3>
+        <div class="underline">&nbsp;</div>
+        <p class="past-projects-p">Let's see our pasts Projects and prepare for incoming conference. Let's Take a look and explore toghther for fun and inspire ideas for them</p>
+        <ul class="past-projects-ul">
+            <li class="past-projects-ul-li"><div class="past-projects-ul-li-div"><span class="past-project-year">2021</span><span class="past-project-text">See our past projects in Seoul</span></div></li>
+            <li class="past-projects-ul-li"><div class="past-projects-ul-li-div"><span class="past-project-year">2020</span><span class="past-project-text">See our past projects in Seoul</span></div></li>
+        </ul>
```
Comment (mckent05) [inline public/pages/about-me.html:84]: - Good job on the work done so far 👏 👏 . There are some differences between this section of your page and the design template. There is a background gradient in each picture. Please check and fix this issue 👍 👍.

Your project:
background grad student


Design Template:
past template

File: public/pages/index.html [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+                </div> 
+                <h3 class="main-program-ul-li-div-h3">CC Ignite</h3>
+                <p class="main-program-ul-li-div-p">Listen to the spakers from various countries about the messages of sharing and opening</p>
+            </div>
+        </li>
+        </ul>
+        <a class="hidden-if-small" id="main-program-full-link" href="#">See the whole Program</a>
+        <button class="hidden-if-large pointer main-program-ul-li-button" type="button"><span class="main-program-ul-li-button-span">Join CC Global Summit 2015</span></button>
+    </section>
+  <section class="featured-speakers">
```
Comment (mckent05) [inline public/pages/index.html]: - The program cards in your design don't exactly match the design template. Please ensure you fix this 😄. Remember it is essential that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project(Please pay attention to the background color of each card):
program cards

Design Template:
program card design

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @heintayzar-hm,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
almost there

## Highlights
- Your site looks great 👍 
- All linter checks are passing correctly ✔️ ✔️ 
- Descriptive PR summary and PR title 👏 👏 


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: public/pages/about-me.html (hunk 6 above)
Comment (heintayzar-hm) [reply public/pages/about-me.html:84]: @mckent05 Do you mean background transparent? I don't understand Overlay or something?

File: public/pages/index.html [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+                </div>
+            </li>
+              <li class="main-program-ul-li">
+                <div class="main-program-ul-li-div">
+                <div class="img-container main-program-ul-li-div-div"><img src="../resources/ignite.png" class="img-logo" alt="cc_ignite">
+                </div> 
+                <h3 class="main-program-ul-li-div-h3">CC Ignite</h3>
+                <p class="main-program-ul-li-div-p">Listen to the spakers from various countries about the messages of sharing and opening</p>
+            </div>
+        </li>
```
Comment (mirouhml) [inline public/pages/index.html]: ## Required

- [ ] Please know that when you hover on a card a white border should appear as per the design:
  image
  In your case this is what happens:
  image
  Kindly fix this 😁.

## Optional

- [ ] For semantic reasons please know that each card should be inside an `article` tag, I suggest you change that as well 😁.

File: public/pages/about-me.html [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
```
Comment (mirouhml) [inline public/pages/about-me.html:84]: - [ ] What the reviewer meant here in this section is that you should match the design provided, in the design there is a red overlay on the pictures, kindly add it, your design is looking like this right now:

  image
  
  And it should be like this 😁: 
  
  image

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @heintayzar-hm, 

Great work on making the changes requested by the previous reviewer 👏 

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet, but don't worry you're almost there!! 😉

image

 ## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @heintayzar-hm       :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for meeting all the project requirements good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**